Assets/Scripts/ItemsScripts/Item_FirstAidKit.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Item_FirstAidKit : MonoBehaviour,Item
+{
+    int _healthToRestore = 50;
+    public void ApplyStatEffects(PlayerStats player)
+    {
+        player.AddToStat("Health", _healthToRestore);
```

Comment:
I think player should just have helth, no need to use some strange method and string for that. Also, thus we can put some custom setter which will kill player on negative hralth

Reply:
Is there another approach to tell the enemy what stats a player has when he shoots? The bullet can transport this information using this dictionary. By using this, the enemy doesn't need access to the whole script of player stats and manually check all parameters of stats that the player stat has. Yes, we can transport whole script with bullet, or have the enemy know this script from spawn, but I also want to use Scriptable Object in a way that we can choose in inspector what stat we want to modify, and set value to modify, with this we don't need to create a big amount of child scripts to item.cs, and don't need to edit scripts at all when we adding new item. 

I'll try to find better way to do this in next commit.

Reply:
I think it is ok to transfer whole script. From the side of performance you anyways just transfer a pointer, not whole class. From the side of access... I also think you can or split script or just don't care, but this dictionary for now only creates additional layer and reduces refactorability (try to rename some string =)).

Reply:
Ok, I am not sure, maybe we can handle it in that way